.kitchen.yml

```diff
@@ -1,22 +1,40 @@
 ---
 driver:
-  name: vagrant
-  vm_hostname: devopsbox.local
+  name: ec2
+  hostname: bastion.local
+driver_config:
+  interface: private
+  instance_type: t2.medium
+  region: us-west-2
+  subnet_id: subnet-XXXXXX
+  security_group_ids: [ "sg-XXXXXX" ]
+  aws_ssh_key_id: <%= ENV['AWS_SSH_KEY_ID'] %>
+  tags:
+    Environment: dev
+    Owner: chef
+    Provisioner: test-kitchen
+    Service: test-kitchen
+    Version: "1.0"
+    Zone: private
+transport:
+  ssh_key: <%= ENV['AWS_SSH_KEY'] %>
 
 provisioner:
   name: chef_zero
-  require_chef_omnibus: true
+  require_chef_omnibus: 12.13.37
 
-# Uncomment the following verifier to leverage Inspec instead of Busser (the
-# default verifier)
-# verifier:
-#   name: inspec
+#verifier:
+#  name: inspec
+#  sudo: true
```

Comment:
I've been filing Github issues to migrate to InSpec. Recommend doing that if you don't have time to migrate.